### PR TITLE
SEO-3260-Core-Description-Too-Short

### PR DESCRIPTION
**Pull request by berylouma**
Hi @Aishwarya-Ganesan 

I have worked on the short descriptions. 

Regards,
Beryl.

### CODE REVIEW

File: aspnet-core/Core-Concepts.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
 ---
 layout: post
-title: Core Concepts | ASP.NET Core | Syncfusion
-description: Core Concepts
+title: Core Concepts in ASP.NET Core Common | Syncfusion
```
Comment (gowthamanthavasiyappan) [inline aspnet-core/Core-Concepts.md:3]: This is not a common control, change in all places.

File: aspnet-core/DataManager/Overview.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 title: Overview | DataManager  | ASP.NET Core | Syncfusion
-description: overview 
+description: Learn here about overview in Syncfusion Essential ASP.NET Core DataManager Control, its elements, and more. 
 platform: aspnet-core
 control: DataManager
 documentation: ug
 ---
 
-# DataManager
+# DataManager in ASP.NET Core
```
Comment (gowthamanthavasiyappan) [inline aspnet-core/DataManager/Overview.md:10]: Change like this - Overview of ASP.NET Core DataManager

File: aspnet-core/GettingStarted/Getting-Started-2-0.md [hunk 3]
```diff
@@ -1,14 +1,14 @@
 ---
 layout: post
 title: Getting Started | ASP.NET Core | Syncfusion
-description: Getting Started.
+description: Learn here more about getting started with Syncfusion Essential ASP.NET Core Common Control, its elements, and more.
```
Comment (gowthamanthavasiyappan) [inline aspnet-core/GettingStarted/Getting-Started-2-0.md:4]: Not common control, check and change in description and h1 tag

**Review by gowthamanthavasiyappan — CHANGES_REQUESTED** [review]

File: aspnet-core/Core-Concepts.md (hunk 1 above)
Comment (berylouma) [reply aspnet-core/Core-Concepts.md:3]: Done

File: aspnet-core/DataManager/Overview.md (hunk 2 above)
Comment (berylouma) [reply aspnet-core/DataManager/Overview.md:10]: Done

File: aspnet-core/GettingStarted/Getting-Started-2-0.md (hunk 3 above)
Comment (berylouma) [reply aspnet-core/GettingStarted/Getting-Started-2-0.md:4]: Done

**Pull request closed without merging**